modules/protect.php

```diff
@@ -407,7 +407,7 @@ function get_headers() {
 		);
 
 		foreach ( $ip_related_headers as $header ) {
-			if ( isset( $_SERVER[ $header ] ) ) {
+			if ( isset( $_SERVER[ $header ] ) && ! empty( $_SERVER[ $header ] ) ) {
```

Comment:
Instead of checking for both `isset` and `empty`, could we simplify this to a single `! empty()` check? `empty` is already checking if the variable is set implicitly.

```suggestion
			if ( ! empty( $_SERVER[ $header ] ) ) {
```